modules/node_groups/node_groups.tf

```diff
@@ -69,6 +69,20 @@ resource "aws_eks_node_group" "workers" {
     }
   }
 
+  dynamic "update_config" {
```

Comment:
Please combine these two dynamic blocks into one.

Reply:
Done @antonbabenko. 
Please checked how I did it, I believe the code is much simpler than the original proposed solution. 
The idea is that by default if you dont add this block on the tf resource it still creates an update_config by default with value of 1.

Tested scenarios:
- max_unavailable_percentage in node_groups_defaults
- max_unavailable in node_groups_defaults
- max_unavailable_percentage in specific node_group (without node_groups_defaults)
- max_unavailable in specific node_group (without node_groups_defaults)
- max_unavailable_percentage in node_groups_defaults with max_unavailable_percentage overwrite in specific node_group 
- max_unavailable_percentage in node_groups_defaults with max_unavailable_percentage overwrite in specific node_group 
- max_unavailable in node_groups_defaults with max_unavailable overwrite in specific node_group 
- max_unavailable in node_groups_defaults with max_unavailable overwrite in specific node_group